src/CoreFungibleToken.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  AccountUpdate,
+  AccountUpdateForest,
+  AccountUpdateTree,
+  Field,
+  PublicKey,
+  UInt64,
+  UInt8,
+  VerificationKey,
+} from 'o1js';
+import { FungibleToken, VKeyMerkleMap } from './FungibleTokenContract.js';
+import {
+  BurnConfig,
+  BurnParams,
+  MintConfig,
+  MintParams,
+  MintDynamicProofConfig,
+  BurnDynamicProofConfig,
+  TransferDynamicProofConfig,
+  UpdatesDynamicProofConfig,
+} from './configs.js';
+
+const defaultMintParams = MintParams.create(MintConfig.default, {
+  minAmount: UInt64.from(1),
+  maxAmount: UInt64.MAXINT(),
+});
+
+const defaultBurnParams = BurnParams.create(BurnConfig.default, {
+  minAmount: UInt64.from(1),
+  maxAmount: UInt64.MAXINT(),
+});
+
+export class CoreToken {
```

Comment:
Can we change one of the examples to use this class, or just annotate a simple example in typedoc comments?